tests/test_benchmark_KPI.py

```diff
@@ -140,15 +140,6 @@ def test_benchmark_Economic_KPI_C2_E2(self, margs):
             sep=",",
             index_col=0,
         )
-        # Define numbers in the csv as int/floats instead of str, but leave row "group" as a string
-        groups = expected_values.loc["group"]
-        # need to transpose the DataFrame before applying the conversion and retranspose after
-        # the conversion because it does not follow the tidy data principle
-        # see https://en.wikipedia.org/wiki/Tidy_data for more info
-        expected_values = expected_values.T.apply(
-            pd.to_numeric, errors="ignore", downcast="integer"
-        ).T
```

Comment:
I thought you transposed the csv so that it is tidy data? I also see that you transpose back and forth, seems to be a trick to get the job done

Reply:
you are commenting on deleted code. I had to introduce this trick of double transpose in the past because I didn't want to perturbe the layout of the data. Now I decided to just make it tidy data :)

Reply:
Ah, okay. I have to learn how to review commit by commit.